### PR TITLE
DVC-5615 - Split the base url override into separate named overrides

### PR DESCRIPTION
**Pull request by JamieSinn**
Fixes the breaking baseURL override

### CODE REVIEW

File: sdk/nodejs/src/eventQueue.ts [hunk 1]
```diff
@@ -36,14 +36,14 @@ export type EventQueueOptions = {
     flushEventQueueSize?: number,
     logger: DVCLogger,
     reporter?: DVCReporter,
-    baseURLOverride?: string
+    eventsAPIBaseURLOverride?: string
```
Comment (ajwootto) [inline sdk/nodejs/src/eventQueue.ts:39]: how is this making it from client initialization into here?

Comment (ajwootto) [reply sdk/nodejs/src/eventQueue.ts:39]: and shouldn't this just use the same name as the outer client option?

Comment (JamieSinn) [reply sdk/nodejs/src/eventQueue.ts:39]: Yeah - missed this in naming updates

Comment (ajwootto) [reply sdk/nodejs/src/eventQueue.ts:39]: it's honestly kind of weird that this class defines its own interface for the options when all of these options are supposed to come directly from the DVCOptions object..

Comment (JamieSinn) [reply sdk/nodejs/src/eventQueue.ts]: 🤷🏻

**Review by ajwootto — APPROVED** [review]

**Pull request merged**